spec/factories/defects.rb

```diff
@@ -7,7 +7,8 @@
     contact_email_address { Faker::Internet.email }
     contact_phone_number { Faker::Base.numerify('###########') }
     trade { Defect::TRADES.sample }
-    target_completion_date { Faker::Date.between(1.day.from_now, 5.days.from_now) }
+    target_completion_date { Faker::Date.between(1.day.ago, 5.days.from_now) }
```

Comment:
Did you intend to leave this in? By doing this, we're now randomly testing in one of three regimes (past, now, future), where there was only one before (future). Is that ok? Does that make the test suite more fragile? Can we use a specific date instead of a random one?

Reply:
I've put this back - it was quite useful for testing things out locally, but it did make tests a bit fragile.